src/sqltables/__init__.py

```diff
@@ -1,2 +1,2 @@
-from .datastructures import SQLTable, SQLTableArg, SQLTableParams
-from .query import SQLTableManager, SQLTableQuery, SQLTableQuerySet
+from .datastructures import SQLTable, SQLTableArg, SQLTableParams  # noqa: F401
```

Comment:
Why the `noqa` directives? I see now that unnecessary imports are checked; are we sure this is right everywhere?